### DISCUSSION TITLE
How To Use Images

### DISCUSSION THREAD

**Discussion by zeltas** [inkle/ink#867] [General]
Can someone post the correct syntax to display an image?
 Thanks

**Comment by JCSTeixeira** [comment]
Hi. It is 
# IMAGE: imagename.jpg

**Reply by zeltas** [reply]
Hi. What I was really looking for is to be able to preview images in Inky as I write script. While this code displays the image in a browser, it doesn't in Inky. But it's strange that stories that are heavily promoted and made by Ink are image heavy.

**Reply by Selsynn** [reply]
I can't speak for the Inkle team of course, but that's most certainly because I think the whole image generation is done on unity side, not in the inky, that is web based tool. 
Web usage of image is totally different from unity and unity way to do is different from unreal. And I think you take any other tool to make a game, and there is more chance it's also another way to handle it. 

Inky is a tool, it's not directly the game engine, it never was the objective of that tool, I think. To be an effective tool, it has to handle a minimum ink syntax. I think it's expecting you to know the rest. 

And it's okay. It's mostly error-free (the only one I complain easily with is the freezing circle loop, but I didn't had it in quite some time. Maybe I also know how to write things better, maybe it's fixed?), it's a free tool, and it helps make great game. It does not create the game for you. 

Honestly, you just have to regenerate your code once and you can see it. Ink is a TEXT tool, not an IMAGE one. I am not surprised the dev and the community is focusing on the TEXT functionnalities, and not to display image. 
It won't help you to know that you have a logical bug to see the image. Seeing the name is well enough when you are testing it. It's even better in my mind, as I can check that it's the right one (the fact I am bad with graphical details may be responsible for that remark.)

**Comment by zeltas** [comment]
Unfortunately that doesn’t work in inky IDE as no image is displayed.


Sent from Yahoo Mail for iPhone


On Monday, October 2, 2023, 8:08 PM, JCSTeixeira ***@***.***> wrote:




Hi. It is

IMAGE: imagename.jpg
